entaxy-project/entaxy
# Feature/blockstack

### Summary

This PR added the following features
- Blockstack Login
- Test Coverage

This PR modified the following features
- Ejected the create-react-app magic, and we have control over all the config files, this is because blockstack needs cors enabled to fetch the manifest file for auth flow.

### Screenshot

![login](https://user-images.githubusercontent.com/1641464/38970553-296b0d72-4364-11e8-91c8-54d32a7c8a8d.gif)

![terminal](https://user-images.githubusercontent.com/1641464/38970555-2c349b22-4364-11e8-860f-30ee56aa8c5c.gif)


### How to test it

`yarn test` to see the test failing because of coverage

`Click the login button` to see your information being pulled from blockstack

### Other Information
I will split PRs into seperate things in the future.
